### ISSUE TITLE
[Bug]: a11y addon scrollbars and completed text overlays highlight checkboxes

### ISSUE THREAD

**Issue by WesselKroos** [storybookjs/storybook#24987]
### Describe the bug

1. When the a11y addon panel has a scrollbar it overlays the highlight checkboxes which prevents you from clicking on a highlight checkbox.

![image](https://github.com/storybookjs/storybook/assets/31220528/7142f841-9a82-4dbe-b680-493e3fd77337)

2. I also have the same issue with the "Tests completed" text that overlays the text and highlight checkbox of the last result when the panel is docked to the right:
![image](https://github.com/storybookjs/storybook/assets/31220528/2e0acc45-5589-47a8-b61e-0a6f31c7445c)

### To Reproduce

1. Open the a11y's Accessibility panel
2. Lower the panel height to make it vertically scrollable
3. Hover your mouse over a highlight checkbox
4. A scrollbar appears which prevents you from clicking on the highlight checkbox

### System

```bash
`npx storybook@latest info` hung after displaying the text "Storybook Environment Info:".
```

### Additional context

It seems to be caused by the `min-width` css property on the `::before` element of the scrollbar. (And I think the `min-height` on the `::before` causes similar issues.)

The second issue can possibly be resolved by adding a padding-bottom to the scrollable content to acommodate for the "Tests completed" text.

**Comment by shilman** [comment]
@cdedreuille is this something you can take a look at? i was easily able to reproduce

**Comment by syedbarimanjan** [comment]
@shilman is this still open can I work on if yes please guide me from where to start.

**Comment by cdedreuille** [comment]
I can take this one on but if you want to give it a try @syedbarimanjan please do. I can assist if you have any issues.

**Comment by syedbarimanjan** [comment]
> I can take this one on but if you want to give it a try @syedbarimanjan please do. I can assist if you have any issues.

thanks alot i will start working on it and wont hesitate to ask for help.

**Comment by Pooya-Oladazimi** [comment]
I cannot find this feature in the Storybook version 9. Is this deprecated? The way to highlight is different now (No checkbox). I cannot find this in the code either.

<img width="1333" height="926" alt="Image" src="https://github.com/user-attachments/assets/477c059b-fa94-41c1-8f8d-207ba8056d4d" />

**Comment by WesselKroos** [comment]
With the smaller scrollbars in Storybook 9 and 10 it seems to be fixed:

[<img width="581" height="249" alt="Image" src="https://github.com/user-attachments/assets/e60b9675-67fa-4892-9917-bcf427788f21" />](https://next--630511d655df72125520f051.chromatic.com/?path=/story/example-page--logged-out)
